Makefile.libretro

```diff
@@ -77,17 +77,32 @@ ifeq ($(USE_DYNAREC),0)
 endif
 
 # iOS
-else ifeq ($(platform),$(filter $(platform),ios-arm64))
+else ifeq ($(platform), ios-arm64)
+	fpic := -fPIC
+	ifeq ($(IOSSDK),)
+		IOSSDK := $(shell xcodebuild -version -sdk iphoneos Path)
+        endif
 	ARCH := arm64
 	USE_DYNAREC = 0
 	HAVE_NEON = 0
 	BUILTIN_GPU = peops
-	TARGET := $(TARGET_NAME)_interpreter_libretro_ios.dylib
+	TARGET := $(TARGET_NAME)_libretro_ios.dylib
```

Comment:
i dont have IOS , but core does not have dynarec for xxx64 builds (yet)... Nondynarec builds are expected to have that filename as is for buildbot to find it

https://github.com/libretro/pcsx_rearmed/blob/master/Makefile.libretro#L82